pkg/cluster/k8sres.go

```diff
@@ -769,7 +769,7 @@ func (c *Cluster) generateSpiloPodEnvVars(uid types.UID, spiloConfiguration stri
 		envVars = append(envVars, customPodEnvVarsList...)
 	}
 
-	if c.OpConfig.WALES3Bucket != "" {
+	if c.OpConfig.WALES3Bucket != "" && enableWALArchiving {
```

Comment:
Unless I'm mistaken, not setting `WAL_S3_BUCKET` will not enable physical backups as well (https://github.com/zalando/spilo/blob/2.0-p7/postgres-appliance/scripts/configure_spilo.py#L862) so I think `enableWALArchiving` should be more generic.

Reply:
And shouldn't we do the same for `c.OpConfig.WALGSBucket` as well?